tests/api/getBaladyiatsForWilaya.test.js

```diff
@@ -0,0 +1,79 @@
+describe('Get baladiyats for wilaya', () => {
+  const mockData = [
+    {
+      mattricule: 33,
+      dairats: [
+        {
+          code: 3301,
+          baladyiats: [
+            {
+              code: 3301,
+            }
+          ]
+        },
+        {
+          code: 3302,
+          baladyiats: [
+            {
+              code: 3302,
+            },
+            {
+              code: 3305,
+            }
+          ]
+        },
+        {
+          code: 3302,
+          baladyiats: [
+            {
+              code: 3302,
+            },
+            {
+              code: 3305,
+            }
+          ]
+        },
+        {
+          code: 3306,
+          baladyiats: [
+            {
+              code: 3303,
+            },
+            {
+              code: 3304,
+            },
+            {
+              code: 3306,
+            }
+          ]
+        }
+      ],
+    }
+  ];
+
+  const baladyiaFiWilaya33 = [
+    { code: 3301 },
+    { code: 3302 },
+    { code: 3305 },
+    { code: 3302 },
+    { code: 3305 },
+    { code: 3303 },
+    { code: 3304 },
+    { code: 3306 }
+  ];
+
+  let getBaladyiatsForWilaya;
+
+
+  beforeEach(()=>{
+    getBaladyiatsForWilaya = require('../../src/api/getBaladyiatsForWilaya');
+  });
+
+  it('should return a curried function that returns data', () => {
```

Comment:
Please add this missing test case ensuring that `getBaladyiatsForWilaya` a function.

```suggestion
  it('should export a function', () => {
    expect(typeof getBaladyiatsForWilaya).toBe('function');
  });

  it('should return a curried function that returns data', () => {
```